src/components/BestSellers/BestSellers.tsx

```diff
@@ -14,31 +14,31 @@ import { motion } from 'framer-motion';
 import { IProduct } from '../../types/productInterfaces';
 import { itemAnimation, textAnimation } from '../../utils/animations';
 import { getProductByParams } from '../../services/products/getProductByParams';
-import {
-  initialSearchParams,
-  FEATURED_CLOTHING_CATEGORY_ID,
-} from '../../constants/constants';
+import { useCategoryData } from '../CategoryDataProvider/CategoryDataProvider';
+import { initialSearchParams } from '../../constants/constants';
 
 import styles from './BestSellers.style';
 
 export const BestSellers: React.FC = () => {
   const [productList, setProductList] = useState<IProduct[]>([]);
+  const { categoryData } = useCategoryData();
+  const idCategory = useMemo(
+    () => categoryData.find((item) => item.key === 'feature')?.id,
+    [categoryData],
+  );
 
   useEffect(() => {
     const fetchData = async () => {
       try {
-        getProductByParams(
-          FEATURED_CLOTHING_CATEGORY_ID,
-          initialSearchParams,
-        ).then((response) => {
+        getProductByParams(idCategory, initialSearchParams).then((response) => {
           setProductList(response.results);
         });
```

Comment:
maybe it's worth not using promises and not making a modern syntax
```suggestion
        const data = await getProductByParams(idCategory, initialSearchParams);
        setProductList(data);
```